eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/actions/ShareActionExecutable.java

```diff
@@ -60,10 +62,24 @@ public void execute(@NonNull Context context)
 
         String eventText = eventText(context);
 
-        Intent sendIntent = new Intent();
-        sendIntent.setAction(Intent.ACTION_SEND);
-        sendIntent.putExtra(Intent.EXTRA_TEXT, eventText);
-        sendIntent.setType("text/plain");
+        Intent sendIntent;
+        if (context instanceof Activity)
```

Comment:
Why do we need this test? Can't we just pass an `Activity` instead of a `Context`? I can't imagine any use case for launching this from something that's not an `Activity`.